### PR TITLE
New @ckeditor/ckeditor5-essentials v27

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by elibarzilay** [comment]
@fedemp: can you please minimize (better: remove) the linter exceptions; also, is `es2015.promise` needed?

File: types/ckeditor__ckeditor5-essentials/tslint.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "ban-types": false,
+        "no-empty-interface": false,
+        "prefer-switch": false,
+        "quotemark": [
+            true,
+            "double",
+            "avoid-escape",
+            "avoid-template"
+        ]
+    }
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-essentials/tslint.json:13]:
```suggestion
    "extends": "dtslint/dt.json"
```
why you're adding those? it passes without exclusions,
```bash
Testing 1 changed packages: ckeditor__ckeditor5-essentials
Testing 1 dependent packages: ckeditor__ckeditor5-essentials
Installing NPM dependencies...
1> ckeditor__ckeditor5-essentials START
2> ckeditor__ckeditor5-essentials START
2> ckeditor__ckeditor5-essentials OK
1> ckeditor__ckeditor5-essentials OK
```

**Comment by fedemp** [comment]
@peterblazejewicz I've force pushed your suggested changes. :+1:

**Review by andrewbranch — APPROVED** [review]

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**